### ISSUE TITLE
[BUG] Segmented Control - Focus outline in Dark theme

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#4278]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When a Segemented Control is used in dark theme where the selected "Tab" is white, the blue focus outline has a visual bug.

## Describe how to reproduce the bug

1. Navigate to the Segmented Control page on the Cookbook
2. Find the section "Colors"
3. Switch to "dark" theme
4. Via keyboard set the selected "tab" in focus

## What was the expected behavior?
Focus outline (on white) should be nice and clean looking:

<img width="136" height="77" alt="Image" src="https://github.com/user-attachments/assets/991b2c32-c06d-4821-a659-0eb4ebdb1402" />

## Add any screenshots
**Cookbook:**
<img width="312" height="107" alt="Image" src="https://github.com/user-attachments/assets/6114cef6-e0c7-457b-919e-07e300e14407" />

## Please complete the following information:
- MacOS
- Chrome
- Version 144.0.7559.133 (Officiel version) (arm64)

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
